core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -261,16 +264,17 @@ private void attachAvailabilitySetParameters(String availabilitySetPrefix, Insta
 
     private Set<InstanceGroup> getInstanceGroups(Set<LoadBalancer> loadBalancers, TargetGroupType type) {
         return loadBalancers.stream()
-            .flatMap(loadBalancer -> loadBalancer.getTargetGroupSet().stream())
-            .filter(targetGroup -> type.equals(targetGroup.getType()))
-            .flatMap(targetGroup -> targetGroup.getInstanceGroups().stream())
-            .collect(Collectors.toSet());
+                .flatMap(loadBalancer -> loadBalancer.getTargetGroupSet().stream())
```

Comment:
can you please remove these tabs? I think there is a cb config in the developer guide on the main page of this gh repo.

Reply:
I do have that configuration set, this change was applied with only the auto-formatter

in the config we specify <setting id="org.eclipse.jdt.core.formatter.continuation_indentation" value="2"/> so this should be correct